site/zenodo_rdm/legacy/deserializers/schemas.py

```diff
@@ -7,168 +7,46 @@
-            identifiers.append({"scheme": "gnd", "identifier": data["gnd"]})
-
-        if identifiers:
-            r["person_or_org"]["identifiers"] = identifiers
-
-        name = HumanName(data["name"])
-        r["person_or_org"]["given_name"] = name.first
-        r["person_or_org"]["family_name"] = name.surnames
-        return r
-
-    def _contributors(self, data):
-        """Load contributors from Zenodo."""
-        if not data:
-            return None
-
-        supervisors = data.get("thesis", {}).get("supervisors", [])
-        if supervisors:
-            for supervisor in supervisors:
-                supervisor["type"] = "Supervisor"
-        contributors = data.get("contributors", [])
-        contributors.extend(supervisors)
-
-        ret = []
-        for c in contributors:
-            contributor = self._creator(c)
-            type = c.get("type")
-            if type:
-                # Zenodo role matches ZenodoRDM ids, lower cased
-                contributor["role"] = {"id": type.lower()}
-            ret.append(contributor)
-
-        return ret
-
-    def _funding(self, grants):
-        """Transform funding information."""
-        _funding = []
-        for grant in grants:
-            # format "10.13039/501100000780::190101904"
-            id_parts = grant["id"].split("::", 1)
-            assert len(id_parts) == 2
-
-            # special case of funder loaded in Zenodo with no DOI related to it.
-            # it is part of the vocabulary.
-            funder_doi = id_parts[0]
-            award_id = id_parts[1]
-            funder_doi_or_ror = FUNDER_DOI_TO_ROR.get(funder_doi, funder_doi)
-
-            _funding.append(
-                {
-                    "funder": {"id": funder_doi_or_ror},
-                    "award": {"id": f"{funder_doi_or_ror}::{award_id}"},
-                }
-            )
-        return _funding
-
-    def _rights(self, data):
-        """Transform rights (licenses)."""
-        if not data:
-            return None
-
-        rdm_license = legacy_to_rdm(data)
-
-        ret = {}
-        if rdm_license:
-            ret = {"id": rdm_license}
-        else:
-            # If license does not exist in RDM, it is added as custom
-            legacy_license = LEGACY_LICENSES.get(data)
-            ret = {"title": {"en": legacy_license["title"]}}
+    class Meta:
+        """Meta class."""
 
-        return [ret]
+        unknown = RAISE
 
-    def _custom_fields(self, data):
+    #         "custom_fields": self._custom_fields(data.get("metadata")),
+    #         # TODO: Map old access rights to RDM access
+    #         "access": self._access(data),
+    #         "files": {"enabled": True},
+    #         "pids": self._pids(data),
+
+    metadata = fields.Nested(MetadataSchema)
+    files = fields.Constant({"enabled": True})
+    # Fields are added on post load
+    custom_fields = fields.List(fields.Dict())
+    access = fields.Dict()
+    pids = fields.Dict()
```

Comment:
minor: we can remove then, so that we don't accidentally accept them as input in the payload